apps/web/src/app/(protectedPage)/forms/[formId]/_components/formEditor/formEditorCard.tsx

```diff
@@ -173,6 +176,23 @@ export function FormEditorCard({ form }: Readonly<Props>) {
     }
   };
 
+  const handleRemoveField = (index: number) => {
+    const removedField = fields[index];
+    remove(index);
+    setState((cs) => ({
+      ...cs,
+      removedFields: [...cs.removedFields, removedField],
+    }));
+  };
+
+  const handleUndoRemoveField = () => {
+    const lastRemovedField = removedFields.pop();
+    if (lastRemovedField) {
+      setState((cs) => ({ ...cs, removedFields }));
+      insert(fields.length, lastRemovedField);
+    }
+  };
```

Comment:
Check for type errors.

**Screenshot**
![image](https://github.com/growupanand/ConvoForm/assets/29487686/7bc9889f-f6a4-4c68-b90e-4ae24e018017)